engines/cge2/vmenu.h

```diff
@@ -76,6 +76,7 @@ class VMenu : public Talk {
 	char *_vmgt;
 	static VMenu *_addr;
 	int _recent;
+	int _lastN = 2;
```

Comment:
This variable is still not initialized in the correct place. This should be done in the `VMenu` constructor implementation (in the menu.cpp file).